.github/workflows/test-linux-gpu.yml

```diff
@@ -0,0 +1,101 @@
+name: Test (Linux/GPU)
+
+on:
+  workflow_dispatch:
+    inputs:
+      build-type:
+        description: Build in Release or Debug?
+        required: true
+        default: Debug
+        type: choice
+        options:
+          - Debug
+          - Release
+  workflow_call:
+    inputs:
+      build-type:
+        required: true
+        default: Debug
+        type: string
+
+jobs:
+  #
+  # Build shards for linux
+  #
+  Linux:
+    name: Build (${{ github.event.inputs.build-type || inputs.build-type }})
+    runs-on: self-hosted
+    outputs:
+      run-tests: ${{ steps.setup.outputs.run-tests }}
+      run-extra-tests: ${{ steps.setup.outputs.run-extra-tests }}
```

Comment:
Those don't exist. If fact there is no need for job outputs in this workflow, since there is only one.

```suggestion
```